### PR TITLE
Update staff pages for March 2024

### PR DESCRIPTION
**Pull request by Laurakko**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/People/Beatmap_Nominators/es.md [hunk 1]
```diff
@@ -51,157 +51,177 @@ Los Beatmap Nominators reciben los siguientes beneficios:
 
 ## Miembros del equipo
 
-Las páginas [Beatmap Nominators](https://osu.ppy.sh/groups/28) y [Beatmap Nominators (Probationary)](https://osu.ppy.sh/groups/32) listan a todos los miembros del equipo.
+The [Beatmap Nominators](https://osu.ppy.sh/groups/28) and [Beatmap Nominators (Probationary)](https://osu.ppy.sh/groups/32) group pages list all of the team members.
 
-Las tablas a continuación indican los modos de juego en donde cada Beatmap Nominator es formalmente calificado para nominar. *Nota: Todos los Beatmap Nominators hablan inglés en adición a los idiomas listados debajo a menos que se especifique otra cosa.*
+The tables listed below indicate the game mode(s) in which each Beatmap Nominator is formally qualified to nominate. *Note: All Beatmap Nominators speak English in addition to the language(s) listed below unless noted otherwise.*
+
+<!-- NOTE FOR TRANSLATORS: Translating this section is different from other parts of the osu! wiki. See https://github.com/ppy/osu-wiki/blob/master/meta/group-info/TRANSLATING.md#group-articles. -->
```
Comment (Walavouchey) [inline wiki/People/Beatmap_Nominators/es.md:58]: revert

File: wiki/People/Beatmap_Nominators/fr.md [hunk 2]
```diff
@@ -52,157 +52,177 @@ Les Beatmap Nominators bénéficient des avantages suivants :
 
 ## Membres de l'équipe
 
-Les pages du groupe [Beatmap Nominators](https://osu.ppy.sh/groups/28) et [Beatmap Nominators (Probationary)](https://osu.ppy.sh/groups/32) présentent la liste de tous les membres de l'équipe.
+The [Beatmap Nominators](https://osu.ppy.sh/groups/28) and [Beatmap Nominators (Probationary)](https://osu.ppy.sh/groups/32) group pages list all of the team members.
 
-Les tableaux ci-dessous indiquent le(s) modes de jeu dans lesquels chaque Beatmap Nominator est formellement qualifié pour proposer une nomination. *Remarque : sauf indication contraire, tous les Beatmap Nominators parlent l'anglais en plus des langues indiquées ci-dessous.*
+The tables listed below indicate the game mode(s) in which each Beatmap Nominator is formally qualified to nominate. *Note: All Beatmap Nominators speak English in addition to the language(s) listed below unless noted otherwise.*
+
+<!-- NOTE FOR TRANSLATORS: Translating this section is different from other parts of the osu! wiki. See https://github.com/ppy/osu-wiki/blob/master/meta/group-info/TRANSLATING.md#group-articles. -->
```
Comment (Walavouchey) [inline wiki/People/Beatmap_Nominators/fr.md:59]: revert

File: wiki/People/Beatmap_Nominators/zh.md [hunk 3]
```diff
@@ -52,42 +52,51 @@ BN 可获得以下特权：
 
 ## 团队成员
 
-[正式 BN 列表](https://osu.ppy.sh/groups/28)以及[见习 BN 列表](https://osu.ppy.sh/groups/32)列出了所有在任的团队成员。
+The [Beatmap Nominators](https://osu.ppy.sh/groups/28) and [Beatmap Nominators (Probationary)](https://osu.ppy.sh/groups/32) group pages list all of the team members.
 
-下表列出了包含所有游戏模式的 BN，他们都具有正式的提名谱面的资格。*注意：除非特殊说明，所有 BN 除了下列语言以外还可以使用英语交流。*
+The tables listed below indicate the game mode(s) in which each Beatmap Nominator is formally qualified to nominate. *Note: All Beatmap Nominators speak English in addition to the language(s) listed below unless noted otherwise.*
+
+<!-- NOTE FOR TRANSLATORS: Translating this section is different from other parts of the osu! wiki. See https://github.com/ppy/osu-wiki/blob/master/meta/group-info/TRANSLATING.md#group-articles. -->
```
Comment (Walavouchey) [inline wiki/People/Beatmap_Nominators/zh.md:59]: revert

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/People/Beatmap_Nominators/en.md [hunk 4]
```diff
@@ -63,36 +63,40 @@ The tables listed below indicate the game mode(s) in which each Beatmap Nominato
...
 | ::{ flag=US }:: [Aeril](https://osu.ppy.sh/users/4334976) |  |
 | ::{ flag=GB }:: [Aistre](https://osu.ppy.sh/users/4879380) |  |
+| ::{ flag=UA }:: [allein](https://osu.ppy.sh/users/6221637) | Ukrainian |
 | ::{ flag=IT }:: [Andrea](https://osu.ppy.sh/users/33599) | Italian |
 | ::{ flag=US }:: [Annabel](https://osu.ppy.sh/users/3388410) | Spanish |
+| ::{ flag=US }:: [Ascended](https://osu.ppy.sh/users/4564285) |  |
 | ::{ flag=SE }:: [bite you death](https://osu.ppy.sh/users/6398464) | Swedish |
+| ::{ flag=US }:: [Bloxi](https://osu.ppy.sh/users/9022451) |  |
 | ::{ flag=PL }:: [browiec](https://osu.ppy.sh/users/9426712) |  |
+| ::{ flag=UK }:: [ckharv](https://osu.ppy.sh/users/9967026) |  |
```
Comment (Walavouchey) [inline wiki/People/Beatmap_Nominators/en.md:77]: wrong flag, should be GB

File: wiki/People/Beatmap_Nominators/en.md [hunk 5]
```diff
@@ -63,36 +63,40 @@ The tables listed below indicate the game mode(s) in which each Beatmap Nominato
...
 | ::{ flag=BR }:: [Malphs](https://osu.ppy.sh/users/9234078) | Portuguese |
 | ::{ flag=CN }:: [MeAqua tete](https://osu.ppy.sh/users/5645691) | Chinese |
-| ::{ flag=RU }:: [Mirash](https://osu.ppy.sh/users/2841009) | Russian |
+| ::{ flag=US }:: [Mimari](https://osu.ppy.sh/users/14339830) |  |
 | ::{ flag=GB }:: [mnyui](https://osu.ppy.sh/users/14261540) |  |
 | ::{ flag=CA }:: [Mochi-](https://osu.ppy.sh/users/9312734) | Chinese |
-| ::{ flag=GB }:: [moonpoint](https://osu.ppy.sh/users/9558549) |  |
-| ::{ flag=TW }:: [Muchin](https://osu.ppy.sh/users/9834516) | Chinese |
+| ::{ flag=CH }:: [momoyo](https://osu.ppy.sh/users/12469536) |  |
+| ::{ flag=TW }:: [MuChin](https://osu.ppy.sh/users/9834516) | Chinese |
```
Comment (Walavouchey) [inline wiki/People/Beatmap_Nominators/en.md:99]: should be Muchin

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/People/Global_Moderation_Team/en.md [hunk 6]
```diff
@@ -192,3 +193,4 @@ The [Global Moderation Team group page](https://osu.ppy.sh/groups/4) lists all o
 
 [^task-Azer]: While not actually being part of the GMT itself, Azer is in the user group by technicality due to needing GMT-level permissions as a World Cup organiser.
 [^task-mangomizer]: While not actually being part of the GMT itself, mangomizer is in the user group by technicality due to needing GMT-level permissions for access to internal forums.
+[^task-mao]: While not actually being part of the GMT itself, mangomizer is in the user group by technicality due to needing GMT-level permissions for access to internal forums.
```
Comment (RoanH) [inline wiki/People/Global_Moderation_Team/en.md:196]: I assume the name needs to be updated:
```suggestion
[^task-mao]: While not actually being part of the GMT itself, Mao is in the user group by technicality due to needing GMT-level permissions for access to internal forums.
```

File: wiki/People/Beatmap_Nominators/en.md (hunk 4 above)
Comment (Laurakko) [reply wiki/People/Beatmap_Nominators/en.md]: fixed

File: wiki/People/Beatmap_Nominators/en.md (hunk 5 above)
Comment (Laurakko) [reply wiki/People/Beatmap_Nominators/en.md]: fixed

File: wiki/People/Global_Moderation_Team/en.md (hunk 6 above)
Comment (Laurakko) [reply wiki/People/Global_Moderation_Team/en.md]: changed to be correct

File: wiki/People/Beatmap_Nominators/es.md [hunk 7]
```diff
@@ -61,147 +61,164 @@ Las tablas a continuación indican los modos de juego en donde cada Beatmap Nomi
...
-| ::{ flag=KR }:: [Beomsan](https://osu.ppy.sh/users/3626063) | Coreano |
-| ::{ flag=PL }:: [browiec](https://osu.ppy.sh/users/9426712) | Polaco |
-| ::{ flag=GB }:: [ckharv](https://osu.ppy.sh/users/9967026) |  |
-| ::{ flag=ES }:: [Didah](https://osu.ppy.sh/users/8030129) | Español |
+| ::{ flag=US }:: [Annabel](https://osu.ppy.sh/users/3388410) | Español |
+| ::{ flag=US }:: [Ascended](https://osu.ppy.sh/users/4564285) |  |
+| ::{ flag=SE }:: [bite you death](https://osu.ppy.sh/users/6398464) | Sueco |
+| ::{ flag=US }:: [Bloxi](https://osu.ppy.sh/users/9022451) |  |
+| ::{ flag=PL }:: [browiec](https://osu.ppy.sh/users/9426712) |  |
+| ::{ flag=UK }:: [ckharv](https://osu.ppy.sh/users/9967026) |  |
```
Comment (Walavouchey) [inline wiki/People/Beatmap_Nominators/es.md:75]:
```suggestion
| ::{ flag=GB }:: [ckharv](https://osu.ppy.sh/users/9967026) |  |
```

File: wiki/People/Beatmap_Nominators/fr.md [hunk 8]
```diff
@@ -62,147 +62,164 @@ Les tableaux ci-dessous indiquent le(s) modes de jeu dans lesquels chaque Beatma
...
-| ::{ flag=KR }:: [Dailycare](https://osu.ppy.sh/users/1634445) | Korean |
-| ::{ flag=PL }:: [Embo](https://osu.ppy.sh/users/10045409) | Polish |
+| ::{ flag=UA }:: [allein](https://osu.ppy.sh/users/6221637) | Ukrainien |
+| ::{ flag=IT }:: [Andrea](https://osu.ppy.sh/users/33599) | Italien |
+| ::{ flag=US }:: [Annabel](https://osu.ppy.sh/users/3388410) | Espagnol |
+| ::{ flag=US }:: [Ascended](https://osu.ppy.sh/users/4564285) |  |
+| ::{ flag=SE }:: [bite you death](https://osu.ppy.sh/users/6398464) | Suédois |
+| ::{ flag=US }:: [Bloxi](https://osu.ppy.sh/users/9022451) |  |
+| ::{ flag=PL }:: [browiec](https://osu.ppy.sh/users/9426712) |  |
+| ::{ flag=UK }:: [ckharv](https://osu.ppy.sh/users/9967026) |  |
```
Comment (Walavouchey) [inline wiki/People/Beatmap_Nominators/fr.md:76]:
```suggestion
| ::{ flag=GB }:: [ckharv](https://osu.ppy.sh/users/9967026) |  |
```

File: wiki/People/Beatmap_Nominators/zh.md [hunk 9]
```diff
@@ -62,32 +62,39 @@ BN 可获得以下特权：
...
 | ::{ flag=US }:: [Aeril](https://osu.ppy.sh/users/4334976) |  |
 | ::{ flag=GB }:: [Aistre](https://osu.ppy.sh/users/4879380) |  |
+| ::{ flag=UA }:: [allein](https://osu.ppy.sh/users/6221637) | 乌克兰语 |
 | ::{ flag=IT }:: [Andrea](https://osu.ppy.sh/users/33599) | 意大利语 |
 | ::{ flag=US }:: [Annabel](https://osu.ppy.sh/users/3388410) | 西班牙语 |
+| ::{ flag=US }:: [Ascended](https://osu.ppy.sh/users/4564285) |  |
+| ::{ flag=SE }:: [bite you death](https://osu.ppy.sh/users/6398464) | 瑞典语 |
+| ::{ flag=US }:: [Bloxi](https://osu.ppy.sh/users/9022451) |  |
+| ::{ flag=PL }:: [browiec](https://osu.ppy.sh/users/9426712) |  |
+| ::{ flag=UK }:: [ckharv](https://osu.ppy.sh/users/9967026) |  |
```
Comment (Walavouchey) [inline wiki/People/Beatmap_Nominators/zh.md:76]:
```suggestion
| ::{ flag=GB }:: [ckharv](https://osu.ppy.sh/users/9967026) |  |
```

File: wiki/People/Beatmap_Nominators/es.md [hunk 10]
```diff
@@ -61,147 +61,164 @@ Las tablas a continuación indican los modos de juego en donde cada Beatmap Nomi
...
+| ::{ flag=BR }:: [Malphs](https://osu.ppy.sh/users/9234078) | Portugués |
+| ::{ flag=CN }:: [MeAqua tete](https://osu.ppy.sh/users/5645691) | Chino |
 | ::{ flag=US }:: [Mimari](https://osu.ppy.sh/users/14339830) |  |
-| ::{ flag=RU }:: [Mirash](https://osu.ppy.sh/users/2841009) | Ruso |
 | ::{ flag=GB }:: [mnyui](https://osu.ppy.sh/users/14261540) |  |
-| ::{ flag=CA }:: [MochiA](https://osu.ppy.sh/users/9312734) |  |
-| ::{ flag=GB }:: [moonpoint](https://osu.ppy.sh/users/9558549) |  |
+| ::{ flag=CA }:: [Mochi-](https://osu.ppy.sh/users/9312734) | Chino |
+| ::{ flag=CH }:: [momoyo](https://osu.ppy.sh/users/12469536) |  |
+| ::{ flag=TW }:: [MuChin](https://osu.ppy.sh/users/9834516) | Chino |
```
Comment (Walavouchey) [inline wiki/People/Beatmap_Nominators/es.md:97]:
```suggestion
| ::{ flag=TW }:: [Muchin](https://osu.ppy.sh/users/9834516) | Chino |
```

File: wiki/People/Beatmap_Nominators/fr.md [hunk 11]
```diff
@@ -62,147 +62,164 @@ Les tableaux ci-dessous indiquent le(s) modes de jeu dans lesquels chaque Beatma
...
 | ::{ flag=GB }:: [mnyui](https://osu.ppy.sh/users/14261540) |  |
-| ::{ flag=CA }:: [Mochi-](https://osu.ppy.sh/users/9312734) | Chinese |
-| ::{ flag=GB }:: [moonpoint](https://osu.ppy.sh/users/9558549) |  |
-| ::{ flag=TW }:: [MuChin](https://osu.ppy.sh/users/9834516) | Chinese |
-| ::{ flag=RO }:: [nanoya](https://osu.ppy.sh/users/12366071) | Romanian |
-| ::{ flag=RU }:: [NeKroMan4ik](https://osu.ppy.sh/users/11387664) | Russian |
-| ::{ flag=SG }:: [neonat](https://osu.ppy.sh/users/1561995) | Chinese |
+| ::{ flag=CA }:: [Mochi-](https://osu.ppy.sh/users/9312734) | Chinois |
+| ::{ flag=CH }:: [momoyo](https://osu.ppy.sh/users/12469536) |  |
+| ::{ flag=TW }:: [MuChin](https://osu.ppy.sh/users/9834516) | Chinois |
```
Comment (Walavouchey) [inline wiki/People/Beatmap_Nominators/fr.md:98]:
```suggestion
| ::{ flag=TW }:: [Muchin](https://osu.ppy.sh/users/9834516) | Chinois |
```

File: wiki/People/Beatmap_Nominators/zh.md [hunk 12]
```diff
@@ -62,32 +62,39 @@ BN 可获得以下特权：
...
 | ::{ flag=CN }:: [Mafumafu](https://osu.ppy.sh/users/3076909) | 汉语 |
 | ::{ flag=BR }:: [Malphs](https://osu.ppy.sh/users/9234078) | 葡萄牙语 |
-| ::{ flag=RU }:: [Mirash](https://osu.ppy.sh/users/2841009) | 俄语 |
+| ::{ flag=CN }:: [MeAqua tete](https://osu.ppy.sh/users/5645691) | 汉语 |
+| ::{ flag=US }:: [Mimari](https://osu.ppy.sh/users/14339830) |  |
 | ::{ flag=GB }:: [mnyui](https://osu.ppy.sh/users/14261540) |  |
 | ::{ flag=CA }:: [Mochi-](https://osu.ppy.sh/users/9312734) | 汉语 |
-| ::{ flag=GB }:: [moonpoint](https://osu.ppy.sh/users/9558549) |  |
+| ::{ flag=CH }:: [momoyo](https://osu.ppy.sh/users/12469536) |  |
 | ::{ flag=TW }:: [MuChin](https://osu.ppy.sh/users/9834516) | 汉语 |
```
Comment (Walavouchey) [inline wiki/People/Beatmap_Nominators/zh.md:98]:
```suggestion
| ::{ flag=TW }:: [Muchin](https://osu.ppy.sh/users/9834516) | 汉语 |
```

File: wiki/People/osu!_Alumni/en.md [hunk 13]
```diff
@@ -10,6 +10,7 @@ The [osu! Alumni group page](https://osu.ppy.sh/groups/16) lists all of the memb
 | :-- | :-- |
 | ::{ flag=CN }:: [\_Stan](https://osu.ppy.sh/users/1653229) | NAT |
 | ::{ flag=IT }:: [-kevincela-](https://osu.ppy.sh/users/266596) | BAT, GMT |
+| ::{ flag=US }:: [-NoName-](https://osu.ppy.sh/users/10400730) |  | GMT |
```
Comment (Walavouchey) [inline wiki/People/osu!_Alumni/en.md:13]: one cell too many, applies to all translations too

```suggestion
| ::{ flag=US }:: [-NoName-](https://osu.ppy.sh/users/10400730) | GMT |
```

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**